Payjoin-spec.md

```diff
@@ -222,6 +222,34 @@ If Alice pays Bob, she might be tempted to pay him a round amount, like `1.23000
 
 For this reason, during a [spare change](#spare-change) situation, we randomly round the amount in the output added by the receiver to the payjoin transaction.
 
+## Risks
+
+### On the receiver side: UTXO probing attack
+
+When the receiver creates a payjoin proposal, he exposes one or several inputs belonging to him.
+
+An attacker could create lot's of original transaction in order to learn the UTXOs of the receiver, while not broadcasting the payjoin proposal.
+
+We cannot fully prevent this type of attack, we implemented the following mitigations:
+
+* When the receiver detects an original transaction being broadcasted, or if the receiver detect that the original transaction has been double spent, then he will reuse the UTXO he exposed for the next payjoin.
+* Only a single payjoin is possible per invoice.
+* While the exposed UTXO will be reused in priority to not leak other UTXOs, there is no strong guarantee about it. This prevent the attacker from detecting with certainty the next payjoin of the merchant to another peer.
+
+Note that probing attack is only a problem for automated payment system like BTCPay Server. End users wallet with payjoin capabilities are not affected, as the attacker can't create multiple invoice to force the receiver to expose his UTXOs.
+
+### On the sender side: Double payment risk for hardware wallets
+
+For a successfull payjoin to happen the sender need to sign two transactions double spending each other: The original transaction and the payjoin proposal.
+
+BTCPay Server or end user wallet can verify that the payjoin proposal is legit by following [the sender's steps explained earlier](#sender).
```

Comment:
`BTCPay Server or end user wallets can verify that the payjoin proposal is legitimate by following [the sender's steps explained earlier](#sender).`